<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Siddhants-MacBook-Air.local" failures="2" tests="7" name="CucumberTests" time="618.888" errors="0" timestamp="2025-09-08T04:02:15 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="127.804"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="135.948">
    <failure type="java.lang.AssertionError" message="A credential (registered email or username) should have been available for the login attempt. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: A credential (registered email or username) should have been available for the login attempt. expected [true] but found [false]
at StepDefinations.CapstoneHomePageSteps.the_login_attempt_should_be_recorded_logged_in_indicator_or_graceful_failure(CapstoneHomePageSteps.java:103)
at ✽.the login attempt should be recorded (logged-in indicator or graceful failure)(file:///Users/siddhantshrivastav/Desktop/Capstone_Project/BDD/src/test/resources/Features/Capstone_HomePage.feature:26)
... Removed 3 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="61.09"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="89.328">
    <failure type="java.lang.AssertionError" message="Tobacco should be visible under Programs expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Tobacco should be visible under Programs expected [true] but found [false]
at StepDefinations.CapstoneHomePageSteps.the_link_should_be_visible_under_programs(CapstoneHomePageSteps.java:135)
at ✽.the "Tobacco" link should be visible under Programs(file:///Users/siddhantshrivastav/Desktop/Capstone_Project/BDD/src/test/resources/Features/Capstone_HomePage.feature:42)
... Removed 3 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="60.626"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="49.491"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="86.066"/>
</testsuite> <!-- CucumberTests -->
